docs/topics/proc_deploying-the-cache-booster-using-the-oc-cli-client.adoc

```diff
@@ -34,7 +35,7 @@ $ unzip {value-name-project}.zip
 $ oc new-project {value-name-project}
 ----
 
-. Navigate to the root directory of your booster.
+. Navigate to the root directory of your project.
```

Comment:
We changed another instance of the same instruction to: Navigate to the root directory of your application.
Please check which more accurate technically, "project" or "application".

Reply:
Agree. I've changed it to applications for the short team and will circulate this question to come to an answer.